package.json

```diff
@@ -64,20 +66,22 @@
   },
   "scripts": {
     "analyze": "yarn build && source-map-explorer 'build/static/js/*.js' --gzip",
-    "build": "yarn install && env $(yarn setenv) vite build && rm build/config.json",
+    "build": "tsc --build && env $(yarn setenv) vite build && rm build/config.json",
```

Comment:
`tsc --build` builds each package and checks types. The packages must be built so that Vite can resolve imports from them.